source/common/filter/config_discovery_impl.h

```diff
@@ -324,14 +338,10 @@ class FilterConfigSubscription
                       const std::string&) override;
   void onConfigUpdateFailed(Config::ConfigUpdateFailureReason reason,
                             const EnvoyException*) override;
+  void updateComplete();
 
   const std::string filter_config_name_;
-  uint64_t last_config_hash_{0ul};
-  ProtobufTypes::MessagePtr last_config_;
-  std::string last_type_url_;
-  std::string last_version_info_;
-  std::string last_factory_name_;
-  SystemTime last_updated_;
+  ConfigVersionSharedPtr last_;
```

Comment:
nit: should this be ConfigVersionConstSharedPtr (I didn't see an update of the `last_` data members)

Reply:
done